config.json

```diff
@@ -1,7 +1,9 @@
 {
   "youtrack_instance_name": "vimc",
-  "youtrack_token": "<PERM_TOKEN>",
+  "youtrack_token": "VAULT:secret/webhook/youtrack:token",
```

Comment:
I wonder (and this is really a special case of https://vimc.myjetbrains.com/youtrack/issue/VIMC-2564) is to consider moving the secret keys from `secret/webhook/...` to `secret/youtrack-integration/...` to be slightly more informative.

Reply:
Good idea